autograder/tests/test_submitty_autograding_shipper.py

```diff
@@ -0,0 +1,152 @@
+import os
+import json
+import shutil
+import sys
+import unittest
+import contextlib
+import copy
+
+# Done to avoid making the base autograder directory a module.
+# This import must be done inside of the class for the mock to take affect
+sys.path.append("..")
+import autograder
+import submitty_autograding_shipper as shipper
+from autograder import config
+
+SCRIPT_DIR = os.path.dirname(os.path.realpath(__file__))
+# Path any auxiliary test files (e.g. autograding configs)
+TEST_DATA_DIR = os.path.join(SCRIPT_DIR, 'data')
+# Equivalent to install dir
+TEST_ENVIRONMENT = os.path.join(SCRIPT_DIR, 'test_environment')
+
+# Holds all system config files (equivalent to /usr/local/submitty/config)
+CONFIG_DIR = os.path.join(TEST_ENVIRONMENT, 'config')
+SUBMITTY_DATA_DIR = os.path.join(TEST_ENVIRONMENT, 'autograding')
+
+# Autograding directories
+TODO_DIR = os.path.join(SUBMITTY_DATA_DIR, 'autograding_TODO')
+DONE_DIR = os.path.join(SUBMITTY_DATA_DIR, 'autograding_DONE')
+TO_BE_GRADED = os.path.join(SUBMITTY_DATA_DIR, 'to_be_graded_queue')
+GRADING = os.path.join(SUBMITTY_DATA_DIR, "grading")
```

Comment:
Nit - this was renamed to `in_progress_grading`

Reply:
Good catch, updated.